include/merlin/core_kernels.cuh

```diff
@@ -90,6 +80,10 @@ __global__ void create_atomic_keys(Bucket<K, V, S>* __restrict buckets,
     for (size_t i = 0; i < bucket_max_size; i++)
       new (buckets[start + tid].keys(i))
           AtomicKey<K>{static_cast<K>(EMPTY_KEY)};
+    K hashed_key = Murmur3HashDevice(static_cast<K>(EMPTY_KEY));
+    uint8_t digest = static_cast<uint8_t>(hashed_key >> 32);
```

Comment:
Could you confirm `uint8_t` should use which heard
[sys/types.h](https://forums.developer.nvidia.com/t/uint8-t-and-sys-types-h/2251)
or <stdint.h>
or https://nvidia.github.io/cutlass/structcutlass_1_1TypeTraits_3_01uint8__t_01_4.html

I mean if we need to add a special header explicitly for `uint8_t`

Reply:
I think  its <cstdint>  or <stdint.h>.